### ISSUE TITLE
Example app is crashing

### ISSUE THREAD

**Issue by chrisjrex** [Raizlabs/RaisinToast#28]
Hi

When I tap on any of the PROTOCOL SUPPORTING VC options in the example app an error is thrown here:

```
- (NSUInteger)supportedInterfaceOrientations
{
    UIViewController *topViewController = [RZRootMessagingViewController topViewController];

    return [topViewController supportedInterfaceOrientations];
}
```

This is in the RZMessagingWindow.m file.

It seems to be hitting an **infinite loop**. If I hard code the returned value then it works fine.

**Comment by adolfo** [comment]
@chrisjrex I've seen this infinite loop as well and my workaround was the same as yours.

**Comment by adamhrz** [comment]
@chrisjrex @adolfo Fixed in #31
